base/i18n/break_iterator_unittest.cc

```diff
@@ -140,9 +140,11 @@ TEST(BreakIteratorTest, BreakWordThai) {
 // dictionary to detect word boundaries in Thai, Chinese, Japanese, Burmese,
 // and Khmer. Due to the size of such a table, the part for Chinese and
 // Japanese is not shipped on mobile.
+// Cobalt does not support Chinese/Japanese word breaking yet. This feature
+// requires a big dictionary(cjdict.txt) to support.
 #if !(BUILDFLAG(IS_IOS) || BUILDFLAG(IS_ANDROID))
 
-TEST(BreakIteratorTest, BreakWordChinese) {
+TEST(BreakIteratorTest, DISABLED_BreakWordChinese) {
```

Comment:
There are actually some discussions happening here: https://github.com/youtube/cobalt/pull/2074. The original idea here is to use DISABLED_ is to better tracking tests. But I don't think that point stands anymore. There will be tests that need to use ``|| defined(STARBOARD)`` to disable parts of it, because they can't be compiled. I think we should have this questions discussed in meetings to keep everyone in the same page.